staging/src/k8s.io/client-go/tools/clientcmd/client_config.go

```diff
@@ -547,7 +547,17 @@ func (config *inClusterClientConfig) Possible() bool {
 // components. Warnings should reflect this usage. If neither masterUrl or kubeconfigPath
 // are passed in we fallback to inClusterConfig. If inClusterConfig fails, we fallback
 // to the default config.
-func BuildConfigFromFlags(masterUrl, kubeconfigPath string) (*restclient.Config, error) {
+func BuildConfigFromFlags(masterUrl string, kubeconfigPath string) (*restclient.Config, error) {
```

Comment:
Hi, we could set `CurrentContext` by accessing `restclient.Config` obj directly.😀I'm afraid we ain't able to provide such helpers for every public fields from `restclient.Config`..

Reply:
using a context in multi-cluster environment is a basic use case for us, but if you want a more generic solution I can rewrite it to take an initialization function as a parameter instead

Reply:
or maybe just passing ConfigOverrides instead of the function, since we don't pass anything into the initializer